Platform/CoffeelakeBoardPkg/Library/Stage2BoardInitLib/Stage2BoardInitLib.c

```diff
@@ -970,9 +970,9 @@ GpioInit (
   GpioTable  = (UINT8 *)AllocateTemporaryMemory (0);  //allocate new buffer
   GpioCfgDataBuffer = GpioTable;
 
-  if (IsPchH()) {
+  if (IsPchH ()) {
     ChipsetId = CNL_H_CHIPSET_ID;
-  } else if (IsPchLp()) {
+  } else {

```

Comment:
I think we should keep IsPchLp().  If it is not H or LP, we should give a fatal error and CpuHalt().

Reply:
Make sense. Let me add CpuHalt() if PCH is not H or LP.

Reply:
To avoid 'uninitialized' variable report, ChipsetId is initialized with CNL_UNKNOWN_CHIPSET_ID.